packages/create-yoga/src/index.ts

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env node
+
+import * as path from 'path'
+import * as fs from 'fs'
+import * as meow from 'meow'
+import * as inquirer from 'inquirer'
+
+import { loadYogaStarter } from './loader'
+import {
+  defaultTemplate,
+  Template,
+  availableTemplates,
+  templatesNames,
+} from './templates'
+
+const cli = meow(
+  `
+    create-yoga [dir]
+
+    > Scaffolds the initial files of your project.
+
+    Options:
+      -t, --template  Select a template. (${templatesNames})
+      --no-install    Skips dependency installation.
+      --no-generate   Skips model generation.
+      --force (-f)    Overwrites existing files.
+`,
+  {
+    flags: {
+      'no-install': {
+        type: 'boolean',
+        default: false,
+      },
+      'no-generate': {
+        type: 'boolean',
+        default: false,
+      },
+      template: {
+        type: 'string',
+        alias: 't',
+        default: false,
+      },
+      force: {
+        type: 'boolean',
+        default: false,
+        alias: 'f',
+      },
+    },
+  },
+)
+
+main(cli)
+
+// Main
+
+async function main(cli: meow.Result) {
+  let template: Template = defaultTemplate
+
+  if (cli.flags['template']) {
+    const selectedTemplate = availableTemplates.find(
+      t => t.name === cli.flags['template'],
+    )
+
+    if (selectedTemplate) {
+      template = selectedTemplate
+    } else {
+      console.log(`Unknown template. Available templates: ${templatesNames}`)
+      return
+    }
+  } else {
+    const res = await inquirer.prompt<{ templateName: string }>([
+      {
+        name: 'templateName',
+        message: 'Choose a GraphQL server template?',
+        type: 'list',
+        choices: availableTemplates.map(t => ({
+          name: `${t.name} (${t.description})`,
+          value: t.name,
+        })),
+      },
+    ])
+
+    template = availableTemplates.find(
+      t => t.name === res.templateName,
+    )
+  }
+
+  let [output] = cli.input
+
+  interface PathResponse {
+    path: string
+  }
+
+  if (!output) {
+    const res = await inquirer.prompt<PathResponse>([
+      {
+        name: 'path',
+        message: 'Where should we scaffold graphql server?',
+        type: 'input',
+      },
+    ])
+
```

Comment:
A default value should be put here in case user do not enter any value (fix was made for graphqlgen recently here https://github.com/prisma/graphqlgen/commit/f382a017b31196dfef782fa7732e634d1002f76e#diff-7ed5e8565b8bbb02f6bf4f39e516cab2)